test/provisioner/provision.go

```diff
@@ -131,11 +131,20 @@ func (p *CloudAPIAdaptor) Delete(ctx context.Context, cfg *envconf.Config) error
 		return err
 	}
 
-	log.Info("Uninstall CoCo and cloud-api-adaptor")
+	log.Info("Uninstall the cloud-api-adaptor")
 	if err = p.installOverlay.Delete(ctx, cfg); err != nil {
 		return err
 	}
 
+	log.Info("Uninstall CoCo")
+	cmd := exec.Command("kubectl", "delete", "-k", "github.com/confidential-containers/operator/config/samples/ccruntime/peer-pods")
```

Comment:
Does the test README need to be updated to require kubectl command to be available on PATH

Reply:
Updated in 475e95862ebca90a8c7fc51c3ee2ffe0e89035d5